### PR TITLE
feat: display an activity indicator in the CLI output when uploading artifacts to cypress cloud

### LINKED ISSUE #28239: Protocol: [Performance] Add a progress indicator for uploads
If protocol uploads take a long time for files, this could cause CI systems to time out. A possible solution to this would be to provide an iterative update to standard out (e.g. dots or something) that at least indicate that progress is being made and triggering CI not to time out

### PR DESCRIPTION
**Pull request by cacieprins**
- Closes [#28239](https://github.com/cypress-io/cypress/issues/28239)

### Additional details

When uploading large artifacts to the cloud, we need a keep alive signal for CI systems, as well as an indicator for users when running in record mode locally.

### Steps to test

1. Throttle network connections using charles proxy or similar (alternatively, select a project that generates a large test replay database or video)
2. Run cypress in record mode
3. Verify the upload indicator progresses

### How has the user experience changed?
Before:
Screen Shot 2023-11-02 at 12 31 40 PM

After:
out_delayed

### PR Tasks

- [X] Have tests been added/updated?
- [NA] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [NA] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: cli/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 13.6.0
```
Comment (cacieprins) [inline cli/CHANGELOG.md]:
```suggestion

## 13.6.0
```

**Review by chrisbreiding — APPROVED** [review]

File: system-tests/lib/normalizeStdout.ts [hunk 2]
```diff
@@ -150,8 +143,8 @@ export const normalizeStdout = function (str: string, options: any = {}) {
   // 15 seconds -> XX seconds
   .replace(/((\d+ minutes?,\s+)?\d+ seconds? *)/g, replaceTime)
   .replace(/\r/g, '')
-  // replaces multiple lines of uploading screenshots & results (since order not guaranteed)
-  .replace(/(Uploading Screenshots & Videos.*?\n\n)((.*-.*[\s\S\r]){2,}?)(\n\n)/g, replaceUploadingResults)
```
Comment (cacieprins) [inline system-tests/lib/normalizeStdout.ts:160]: This mask was no longer in used, so replaced with the activity mask

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
The animation looks great.

@cacieprins Could we have the 'Uploading' text indented 2 spaces like the other text? Maybe in the same Blue style?

Screen Shot 2023-11-14 at 9 55 22 AM

I also wonder if there should be some interval before printing the 'Uploading Cloud Artifacts: .' text, where it will just skip this if it uploads so fast. Most of my run just had this line printed arbitrarily when the upload was instant.

**Comment by jennifer-shehane** [comment]
@cacieprins Mine has the Uploading displaying without a spaced line above it from the Test Replay output, not like shown in your screenshots.

Screen Shot 2023-11-16 at 11 35 00 AM

**Review by jennifer-shehane — APPROVED** [review]

**Pull request merged**